src/pages/detail.tsx

```diff
@@ -208,52 +208,56 @@ export default function SessionDetail() {
                 )}
               </Row>
             </DetailContainer>
-            <Text variant="h4" mt={8} mb={6} color="neutral.c100">
-              {t("sessions.detail.accounts")}
-            </Text>
 
-            <CustomList>
-              {Array.from(sessionAccounts).map(([chain, accounts]) => {
-                return (
-                  <Box key={chain} mb={6} flex={1}>
-                    <Box mb={6}>
-                      <Text variant="subtitle" color="neutral.c70">
-                        {getDisplayName(chain)}
-                      </Text>
-                    </Box>
+            {Array.from(sessionAccounts).length > 0 ? (
```

Comment:
Could you move `getAccountsFromAddresses` function outside of the component and return directly an array from `sessionAccounts`, please?
Today the `useMemo` dependencies do not reflect the reality as accounts is not used by the function `getAccountsFromAddresses`. By using it as a second parameter you can extract the function from the component (will not be redefined for each render) and the dependencies list of `useMemo` reflects what you use inside.

Reply:
Done ✅